docs/fabrication/hand-tools/punches.md

```diff
@@ -0,0 +1,14 @@
+# Punches
+## Using Punches
+### Proper Grip and Handling
+## Punch Functions
+### Creating Precise Holes and Indents
+### How to Use Punches
+#### Holding the Punch
+#### Aligning with the Mark
+#### Applying Controlled Force
+#### Techniques for Different Types of Punches
+## Different Types of Punches
+### Center Punches
+### Drive Pin Punches
```

Comment:
Dunno if the following under are super applicable. Imo this whole page could be shorter